netcore/Makefile

```diff
@@ -47,6 +48,13 @@ TEST_ASSETS_URL = $(FEED_BASE_URL)/corefx-tests/$(NETCORETESTS_VERSION)/$(TESTS_
 BLAME = $(shell git blame ../configure.ac | grep AC_INIT | cut -f1 -d' ')
 VERSTUB = .$(shell git rev-list --count $(BLAME)..HEAD)
 
+all: runtime bcl
+
+LOCAL_DOTNET = ../.dotnet/dotnet
+$(LOCAL_DOTNET):
+	bash ../eng/common/dotnet-install.sh -v $(NETCOREAPP_VERSION) && \
```

Comment:
If we do something like:

```bash
bash -l -c ". ../eng/common/tools.sh && InitializeToolset"
```

We'll be running the same tools install as corefx does.

Reply:
Never-mind, it seems like there's some unknowns cherry picking arcade functions.